src/test/java/interface_adapter/recipeCollection/RecipeCollectionPresenterTest.java

```diff
@@ -0,0 +1,36 @@
+package interface_adapter.recipeCollection;
+
+import data_access.InMemoryRecipeCollectionDAO;
+import org.junit.jupiter.api.Test;
+import use_case.recipeCollection.RecipeCollectionInteractor;
+import use_case.recipeCollection.RecipeCollectionOutputBoundary;
+import use_case.recipeCollection.RecipeCollectionOutputData;
+
+import static org.junit.jupiter.api.Assertions.*;
+
```

Comment:
Add Javadocs here!

Reply:
This will be significant especially these are for tests, and an explanation will be very helpful in understanding what you are testing.